packages/core/lifecycle/core-lifecycle-server-internal/kibana.jsonc

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "shared-common",
```

Comment:
The package type in `kibana.jsonc` is auto-generated and might change at a later stage if and when different package types are introduced.